README.md

```diff
@@ -58,8 +58,7 @@ All contents of source\_folder will be copied into the disk image.
 *   **--rez \<rez_path\>:** specify custom path to Rez tool used to include license file
 *   **--no-internet-enable:** disable automatic mount&copy
 *   **--format:** specify the final image format (default is UDZO)
-*   **--add-file \<target_name\> \<path_to_source_file\> \<x\> \<y\>:** add additional file (option can be used multiple times)
-*   **--add-folder \<target_name\> \<path_to_source_folder\> \<x\> \<y\>:** add additional folder (option can be used multiple times)
+*   **--add-file \<target_name\> \<path_to_source_file\> \<x\> \<y\>:** add additional file or folder (can be used multiple times)
```

Comment:
maybe `--add-resource` and `<path_to_resource>` or simply `<path>`?

Reply:
I think "resource" is the wrong term to use here; that has more specific meanings in the contexts of various computing things, especially on Mac, where there's the whole "resource fork" thing, which is a super-special form of alternate file data stream.

Maybe just change `path_to_source_file` to `file | folder` to indicate that you can use either?